src/Client/MultiplexedConnections.cpp

```diff
@@ -214,9 +222,10 @@ Packet MultiplexedConnections::drain()
     Packet res;
     res.type = Protocol::Server::EndOfStream;
 
+    static constexpr UInt64 draining_timeout_seconds = 3;
```

Comment:
Maybe we can get this value from server config?
I don't like constants in the middle of the code ...
Also, it is not clear why we use `3`